composer/trainer/trainer.py

```diff
@@ -1018,9 +1146,150 @@ def saved_checkpoints(self) -> List[Tuple[Timestamp, List[pathlib.Path]]]:
             return []
         return self._checkpoint_saver.saved_checkpoints
 
-    def fit(self):
-        """Train and evaluate the model on the provided data."""
-        # Print any exception, so it can be caputred by any callbacks or loggers (e.g. WandB, FileLogger)
+    def fit(
+        self,
+        *,
+        # Train Dataloader
+        train_dataloader: Optional[Union[Iterable, DataSpec, Dict[str, Any]]] = None,
+        train_dataloader_label: str = "train",
```

Comment:
is this new? what is the use case for this label?

Reply:
This is if you want to train on seperate datasets via multiple calls to fit, and track metrics separately (since metrics are indexed by the dataloader label)